api/opencl_runtime_layer.asciidoc

```diff
@@ -228,7 +228,7 @@ Otherwise, it returns one of the following errors:
   * {CL_INVALID_CONTEXT} if _context_ is not a valid context.
   * {CL_INVALID_DEVICE} if _device_ is not a valid device or is not associated
     with _context_.
-  * {CL_INVALID_OPERATION} if _device_ does not support On-Device Queues.
+  * {CL_INVALID_OPERATION} if _device_ does not support a replaceable default On-Device Queue.
```

Comment:
The new error condition is good but is it worth just listing both?   Doesn't actually change anything, just saves thinking about the transitive property that all devices that do not support on-device queues won't support also won't support replacing the default.

Reply:
@bashbaug  and I went back and forth on this and decided to keep just one. I'll leave it as single condition for now, unless multiple reviewers want to list both.

Reply:
Ah, sorry, I missed the earlier discussion on this point.